src/construct.yaml

```diff
@@ -0,0 +1,25 @@
+{% from "version.jinja" import version %}
+{% from "python.jinja" import py_ver_maj, py_ver_min %}
+
+
+name: miniforge-py{{ py_ver_maj }}{{ py_ver_min }}
+version: {{ version }}
+
+install_in_dependency_order: true
+
+channels:
+  - http://conda.anaconda.org/conda-forge
+  - https://repo.continuum.io/pkgs/main
+  - https://repo.continuum.io/pkgs/free
+  - https://repo.continuum.io/pkgs/r
+  - https://repo.continuum.io/pkgs/pro
+  - https://repo.continuum.io/pkgs/msys2
+
+specs:
+  - python {{ py_ver_maj }}.{{ py_ver_min }}.*
+  - pip
```

Comment:
Require `pip` explicitly. This is technically not needed if the `condarc` value `add_pip_as_python_dependency` is `true` (default). However if `add_pip_as_python_dependency` is `false`, this needs to be explicitly added. So we simply always require `pip` to ensure it is available for the end user.